common/src/main/resources/application.common.properties

```diff
@@ -1,3 +1,5 @@
 efs.mount=${AB2D_EFS_MOUNT:#{systemProperties['java.io.tmpdir']+'/jobdownloads'}}/
 
-audit.files.ttl.hours=24
\ No newline at end of file
+audit.files.ttl.hours=24
+
+hpms.base.url=https://hpmsimpl.cms.gov
```

Comment:
Make this an environment variable with this value as a default. Environment variables start with AB2D_

Reply:
Done